mux_test.go

```diff
@@ -35,8 +35,10 @@ type routeTest struct {
 	path           string            // the expected path of the match
 	pathTemplate   string            // the expected path template to match
 	hostTemplate   string            // the expected host template to match
+	additionalPath string            // additional path to add to route (and parse) during test
```

Comment:
Can this not be rolled into an existing field for testing? (just wary of adding once-off test fields)